content/install_maintain/installation/installing-cluster.md

```diff
@@ -76,6 +76,43 @@ cfy cluster update-profile
  - Make sure the PostgreSQL instance is publicly available and reachable from the local Cloudify Manager machine.
  - Download the CA certificate installed on the PostgreSQL DB instance into the local Cloudify Manager machine.
  
+##### Azure DBaaS for Postgres
+
+Azure has support for DBaaS for Postgres.  
```

Comment:
Cloudify supports Microsoft's [Azure Database for Postgres](https://docs.microsoft.com/en-us/azure/postgresql/) as an external database option replacing Cloudify's PostgreSQL deployment.

Azure Database for Postgres is a fully managed database-as-a-service offering that can handle mission-critical workloads with predictable performance, security, high availability, and dynamic scalability. It is available in two deployment options, as a single server and as a Hyperscale (Citus) cluster (preview). 

######Setting up Azure database for PostgreSQL as the Cloudify database